components/expenses/ExpenseFormPayeeSignUpStep.js

```diff
@@ -124,21 +139,39 @@ const ExpenseFormPayeeSignUpStep = ({ formik, collective, onCancel, onNext }) =>
 
   const setPayoutMethod = React.useCallback(({ value }) => formik.setFieldValue('payoutMethod', value), []);
   const [payeeType, setPayeeType] = React.useState(PAYEE_TYPE.USER);
+  const [validateSlug, { data: existingSlugAccount }] = useLazyQuery(validateSlugQuery, {
+    context: API_V2_CONTEXT,
+  });
+
   const changePayeeType = e => {
     e.stopPropagation();
     setPayeeType(e.target.value);
   };
 
   React.useEffect(() => {
     if (values.payee?.organization?.name) {
-      formik.setFieldValue('payee.organization.slug', suggestSlug(values.payee.organization.name));
+      const slug = suggestSlug(values.payee.organization.name);
+      if (values.payee.organization.slug !== slug) {
+        formik.setFieldValue('payee.organization.slug', suggestSlug(values.payee.organization.name));
```

Comment:
This should only be done if the slug hasn't been touched. If I put something in the `slug` input then later go and edit the name, the slug I've set should not be overwritten.